installer/roles/kubernetes/templates/postgresql-persistent.yml.j2

```diff
@@ -159,18 +159,18 @@ parameters:
 - description: Username for PostgreSQL user that will be used for accessing the
     database.
   displayName: PostgreSQL Connection Username
-  from: user[A-Z0-9]{3}
+  from: {{ pg_username }}
```

Comment:
Does this actually fix the problem? Unless I'm missing something, these are still going to show up in the environment: https://github.com/ansible/awx/blob/b12d852e6d1cf491c77d44aff3fb62bf5b997003/installer/roles/kubernetes/templates/postgresql-persistent.yml.j2#L90-L94

Reply:
It does fix the problem. The problem was two fold. 1. the environment secure variable gets overwritten by the shell command that follows which has been removed. 2. Adding the Ansible variable to the jinja2 lets the secret get created correctly and remain. Today the postgresql secret sits there doing nothing with the default values.

Reply:
The variable set in the postgresql template you cited is inherited from the variable that I have changed.

Reply:
Thanks for the explanation. I will pull this down and test it out, probably Monday, and merge if things go well.